examples/platform/linux/DeviceAttestationCredsEle.cpp

```diff
@@ -0,0 +1,223 @@
+/*
+ *
+ *    Copyright (c) 2021-2022 Project CHIP Authors
+ *    Copyright 2023, 2025 NXP
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+#include "DeviceAttestationCredsEle.h"
+
+#include <crypto/CHIPCryptoPAL.h>
+#include <lib/core/CHIPError.h>
+#include <lib/support/Span.h>
+
+/* Keystore ID inside the ELE */
+constexpr int KEY_STORE_ID = 0xBBBB
+    /* Authentication nonce */
+    constexpr int AUTHEN_NONCE = 0x2222
```

Comment:
Alignment suggests this does not compile (missing at least `;`).

Do not check in code that does not even compile.